consultation_analyser/consultations/models.py

```diff
@@ -1,7 +1,51 @@
-# Create your models here.
+import uuid
 from django.db import models
 
 
-class Question(models.Model):
+class UUIDPrimaryKeyBase(models.Model):
```

Comment:
And let's also timestamp all these models!

Reply:
Have added a timestamp model.